tests/jax/components/training/step_test.py

```diff
@@ -0,0 +1,475 @@
+                            ],
+                            [
+                                [0.1, 0.5, 0.7, 0.1, 0.5, 0.7],
+                                [0.1, 0.5, 0.7, 0.1, 0.5, 0.7],
+                            ],
+                        ]
+                    ),
+                    legal_actions=jnp.array(
+                        [
+                            [
+                                [1, 1, 0, 1, 0, 0],
+                                [0, 1, 0, 0, 1, 0],
+                            ],
+                            [
+                                [0, 1, 1, 1, 0, 1],
+                                [1, 0, 1, 0, 1, 0],
+                            ],
+                        ]
+                    ),
+                    terminal=jnp.array(
+                        [
+                            [
+                                [0, 0, 0, 0, 0, 1],
+                                [0, 0, 0, 0, 0, 0],
+                            ],
+                            [
+                                [0, 1, 1, 0, 0, 1],
+                                [0, 0, 0, 1, 1, 0],
+                            ],
+                        ]
+                    ),
+                ),
+                "agent_2": OLT(
+                    observation=jnp.array(
+                        [
+                            [
+                                [0.1, 0.5, 0.7, 0.9, 0.9, 0.8],
+                                [0.1, 0.5, 0.7, 0.9, 0.9, 0.8],
+                            ],
+                            [
+                                [0.1, 0.5, 0.7, 0.1, 0.5, 0.7],
+                                [0.1, 0.5, 0.7, 0.1, 0.5, 0.7],
+                            ],
+                        ]
+                    ),
+                    legal_actions=jnp.array(
+                        [
+                            [
+                                [1, 1, 0, 1, 0, 0],
+                                [0, 1, 1, 0, 0, 0],
+                            ],
+                            [
+                                [0, 0, 0, 1, 0, 1],
+                                [0, 0, 1, 0, 1, 0],
+                            ],
+                        ]
+                    ),
+                    terminal=jnp.array(
+                        [
+                            [
+                                [1, 0, 0, 0, 0, 1],
+                                [0, 1, 0, 0, 0, 0],
+                            ],
+                            [
+                                [1, 1, 1, 0, 1, 1],
+                                [0, 0, 0, 1, 1, 0],
+                            ],
+                        ]
+                    ),
+                ),
+            },
+            actions={
+                "agent_0": jnp.array(
+                    [
+                        [[1, 2, 1, 1, 3, 1], [4, 2, 0, 0, 0, 0]],
+                        [[1, 1, 1, 1, 2, 1], [1, 3, 0, 4, 0, 0]],
+                    ]
+                ),
```

Comment:
Again, according to the output data from the example on the debug environment it seems that `actions` should take on the following form 
```
"agent_0": jnp.array(
                    [
                        [[1, 2, 1,], [4, 2, 0, ]],
                        [[1, 1, 1,], [1, 3, 0,]],
                        [[1, 2, 1], [2, 1, 0,]], 
                    ]
                ),
```